src/pdfviewer/PDFDocument.cpp

```diff
@@ -2733,7 +2747,21 @@ void PDFDocument::setupMenus(bool embedded)
 
     configManager->modifyManagedShortcuts("pdf");
 }
-
+/*
+ * Copies to clipboard position in cm captured from the pdf viewer (w.r.t. south west corner)
+ */
+void PDFDocument::getPosFromCLick(const QPointF &pos)
+{
+	int page = pdfWidget->pageFromPos(pos.toPoint());
+	if (page < 0) return;
+	float height = (pdfWidget->pageRect(page).height()/pdfWidget->totalScaleFactor()) / 28.45;
+	QClipboard *clipboard = QGuiApplication::clipboard();
```

Comment:
are you sure about the 28.45 ?
I assume 72/inch = ?

Reply:
Hey! No, I'm not sure... I found this approximate conversion from pt to cm online and it worked just fine... Sorry I didn't get your suggestion, how is it?


By do way, any suggestions on how to document this feature?

Best!
Lucas

Reply:
1pt = 1inch/72.27 = 2.54cm/72.27. So x pt = x / (1/(1pt)) = ( x / (72.27/2.54) )cm where 72.27/2.54 = 28,452756

Reply:
If pt is postscript point not TeX point then use 72.00 instead of 72.27. Hence result is 28.346457.

Reply:
poppler says that pageSize is given in points (i.e. 1/72th of an inch)

Reply:
Thanks! I pushed a new commit with this fix on the conversion. I tested it by picking points from a grid that I drew and the results are way more precise :)